python/bindings/sidekiq_rx_python.cc

```diff
@@ -37,6 +37,9 @@ void bind_sidekiq_rx(py::module& m)
         std::shared_ptr<sidekiq_rx>>(m, "sidekiq_rx", D(sidekiq_rx))
 
         .def(py::init(&sidekiq_rx::make),
+           py::arg("_card"),
+           py::arg("port_id"),
```

Comment:
should this be named `port_id1` since there is a `port_id2`?

Reply:
This is generated code, but there was an issue in sidekiq_base.h that is fixed